packages/react-composites/src/composites/ChatComposite/ImageUpload/useImageUpload.tsx

```diff
@@ -148,7 +119,11 @@ export const useImageUpload = (): [
       handleInlineImageUploadAction({ type: AttachmentUploadActionType.Remove, id: uploadId });
       // TODO: remove local blob
       if (imageUpload?.metadata.progress === 1) {
-        adapter.deleteImage(imageId);
+        try {
+          adapter.deleteImage(imageId);
+        } catch (error) {
+          console.error(error);
```

Comment:
Do we need to also have have this?

<img width="805" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/36b04382-c814-4f98-8030-b7c27925ca67">

Reply:
@palatter This is for deleting the image from the server. This action is kind of optional since the image will be auto-deleted after 7 days. If the deleting call fails, I think we don't need to show an error bar to the end user but just show a console log to the Contoso developer.